src/utils/jetifier.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path';
+import chalk from 'chalk';
+import { execFileSync } from 'child_process';
+import fs from 'fs-extra';
+import { Report } from '../types';
+
+type Options = {
+  root: string;
+  input: string;
+  output: string;
+  reverse: boolean;
+  report: Report;
+};
+
+export default async function jetifier({ root, input, output, reverse }: Options) {
+  const jetifierStandalone = path.join(root, 'node_modules', '.bin', 'jetifier-standalone')
+
+  if (await fs.pathExists(jetifierStandalone)) {
+    const args = ['-i', input, '-o', output];
```

Comment:
You should also include jetifier as an optionalDependency

Reply:
Added 👍